src/MoBi.Presentation/Tasks/ObservedDataTask.cs

```diff
@@ -257,7 +257,7 @@ private IEnumerable<ColumnInfo> createColumnInfos()
             NullValuesHandling = NullValuesHandlingType.DeleteRow,
          };
 
-         timeColumn.DimensionInfos.Add(new DimensionInfo {Dimension = timeDimension, IsMainDimension = true});
+         timeColumn.DimensionInfos.Add(new DimensionInfo.DimensionInfo {Dimension = timeDimension, IsMainDimension = true});
```

Comment:
@georgeDaskalakis  Why do we have TWO DimensionInfo? Isn't it weird? I'll comment it just to have a discussion